website/content/en/docs/concepts/nodeclasses.md

```diff
@@ -280,7 +280,6 @@ The Bottlerocket AMIFamily does not support the following fields:
 * `evictionSoft`
 * `evictionSoftGracePeriod`
 * `evictionMaxPodGracePeriod`
-* `cpuCFSQuota`
```

Comment:
I think we could probably extend this to all docs versions -- looks like we have supported this value for quite some time.

Reply:
Agreed, but we never explicitly called this out in any other docs except for v1 and preview.

Reply:
Oh -- didn't realize that this was an addition to the docs. Makes sense. Thanks for catching this.